### DISCUSSION TITLE
Graphical planner for future SAT passes

### DISCUSSION THREAD

**Discussion by oh1jw** [magicbug/Cloudlog#2995] [Ideas / Feature Requests]
An unnamed logging program has a design tool that can visualize Radar-type satellite passes.
Something similar could be built on top of that Satellite Timers functionality. 
I think it would be quite handy when planning Rover activities. 

You could generate a visualization for many upcoming passes, maybe export them to images or PDF -file, 
or internal shortcut link, like ../index.php/sattimers/20240227/

@magicbug @phl0

**Comment by phl0** [comment]
What logging program is that pls? Is it open source?
